src/components/SearchResult.tsx

```diff
@@ -0,0 +1,47 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import styles from './SearchResult.module.scss';
+
+interface ISearchResult {
+  className?: string;
+  snippet: string;
+  title: string;
+  url: string;
+}
+
+const SearchResult: React.FC<ISearchResult> = (props) => (
+  <div
+    className={props.className}
+  >
+    <h3>
```

Comment:
My understanding is that the article listing page uses an h3 element, and then we use the h3 element here for each result. I think we this can be an h2 element.